crate/cli/src/actions/shared/import_key.rs

```diff
@@ -15,7 +15,10 @@ use cosmian_kms_client::{
 use zeroize::Zeroizing;
 
 use super::utils::{build_usage_mask_from_key_usage, KeyUsage};
-use crate::{actions::console, error::CliError};
+use crate::{
+    actions::console,
+    error::{result::CliResult, CliError},
```

Comment:
same here

Reply:
indeed. Done in https://github.com/Cosmian/kms/commit/177df68c37757fcbabe92f97d8ceac02e7160299.